model/pipeline/src/main/proto/schema.proto

```diff
@@ -87,4 +87,56 @@ message LogicalType {
   string urn = 1;
   bytes payload = 2;
   FieldType representation = 3;
+  FieldType argument_type = 4;
+  FieldValue argument = 5;
```

Comment:
This was done as a continuation of the previous PR that was already merged - the changes made in that PR were not reflected in the portability protos get. Since the proto is used as the arg for the OneOf, we needed it to be reflected in the proto for completeness.

This is also done to enable your option work. I think making the payload binary there is the wrong approach - part of the point of the proto is to be easily introspectible (i.e. you should be able to see the option value by simply examining the proto, with no need to parse a binary option value).